base/templates/download.html

```diff
@@ -0,0 +1,20 @@
+{% extends "_layouts/default.html" %}
+
+{% block content %}
+
+<h2>
+<a id='download-link' href='{{ signed_download_url }}'>{{ title }}</a> download will begin shortly...
+</h2>
+
+{% endblock %}
+
+{% block script %}
+
+<script>
+$(document).ready(function() {
+  $('#download-link').get(0).click();
```

Comment:
This is probably a good idea. There is a 'download_bams_script.sh' page that no longer works:

https://development-1-6-0-dot-andersen-lab.appspot.com/data/release/20210121/download/download_strain_bams.sh

We can either (1) have users request access or (2) allow users to download with a 'requester pays' option setup on GCP. This would allow users to download the data but they would have to pay for the download costs.